drivers/kscan/Kconfig.xec

```diff
@@ -45,3 +46,46 @@ config KSCAN_XEC_POLL_PERIOD
 	  Defines the poll period in msecs between between matrix scans.
 
 endif # KSCAN_XEC
+
+menuconfig KSCAN_XEC_V2
+	bool "XEC Microchip KSCAN V2 driver"
+	depends on SOC_SERIES_MEC172X
+	select MULTITHREADING
+	help
+	  Enable the Microchip XEC MEC172x Kscan IO driver.
+
+if KSCAN_XEC_V2
+
+config KSCAN_XEC_COLUMN_SIZE
+	int "KSCAN_XEC_COLUMN_SIZE"
+	default 16
+	help
+	  Adjust the value to your keyboard columns. The maximum
+	  column size for the Microchip XEC family is 18 (from 0 to 17).
+
+config KSCAN_XEC_ROW_SIZE
+	int "KSCAN_XEC_ROW_SIZE"
+	default 8
+	help
+	  Adjust the value to your keyboard rows. The maximum
+	  column size for the Microchip XEC family is 8 (from 0 to 7).
+
+config KSCAN_XEC_DEBOUNCE_DOWN
+	int "KSCAN_XEC_DEBOUNCE_DOWN"
```

Comment:
Why is the prompt a copy of the setting name, and not a real prompt text ?

Reply:
updated with prompt text